actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -174,6 +191,15 @@ def initialize_copy(other)
       end
     end
 
+    # Specify whether to prevent the user agent from loading any assets over
+    # HTTP when the page uses HTTPS:
+    #
+    #   policy.block_all_mixed_content
+    #
+    # Pass +false+ to allow it again:
+    #
+    #   policy.block_all_mixed_content false
+    #
```

Comment:
@jonathanhefner Do you think this blank line should be deleted?
I'm seeing both for comments ending with single line code blocks.

Reply:
I don't know that we have policy either way.  Personally, I think it looks nicer to have it (and for multi-line code blocks, too). :+1: